src/decorator.rb

```diff
@@ -0,0 +1,10 @@
+require './nameable'
```

Comment:
> - [ ] Please I suggest you use require_relative instead of require as the file path throws an error when checking the result.👍
> 
> ![tu](https://github.com/fatima-najafi/oop-school-library/assets/40009354/cf0aa9e4-5e68-4dd5-92a3-311fe2b2b1f0)
> 
> ```suggestion
> require_relative 'nameable'
> ```